firebase-sessions/src/test/kotlin/com/google/firebase/sessions/testing/TestSessionEventData.kt

```diff
@@ -45,7 +45,7 @@ internal object TestSessionEventData {
       applicationInfo =
         ApplicationInfo(
           appId = FakeFirebaseApp.MOCK_APP_ID,
-          deviceModel = "",
```

Comment:
Do the tests have access to `Build.MODEL` too?

Reply:
Seems they do - updated